rllib/examples/policy/cliff_walking_wall_policy.py

```diff
@@ -0,0 +1,85 @@
+from ray.rllib.policy.policy import Policy, ViewRequirement
```

Comment:
Do you think you can implement a unittest for this policy? One thing that I can think of is to check if the returned reward by this policy matches the expected reward from all possible starting locations in the env?